src/Driver/SolverTypes/IMEXSolverType.jl

```diff
@@ -0,0 +1,160 @@
+
+export IMEXSolverType
+
+"""
+# Description
+    IMEXSolverType(;
+        splitting_type = HEVISplitting(),
+        linear_model = AtmosAcousticGravityLinearModel,
+        implicit_solver = ManyColumnLU,
+        implicit_solver_adjustable = false,
+        solver_method = ARK2GiraldoKellyConstantinescu,
+        solver_storage_variant = LowStorageVariant(),
+    )
+
+This solver type constructs a solver for ODEs with the
+additively-partitioned form:
+
+```math
+  \\dot{Q} = [l(Q, t)] + [f(Q, t) - l(Q, t)]
+```
+
+where `Q` is the state, `f` is the full tendency and `l` is the chosen
+implicit operator.
+
+# Arguments
+- `splitting_type` (DiscreteSplittingType): The type of discrete
+    splitting to apply to the right-hand side.
+    Default: `SlowFastSplitting()`
```

Comment:
Doesn't match the actual default.

Reply:
Yes that was my bad! I was copy-pasting parts of docstrings I previously wrote and wasn't careful here!